interface/patient_file/ccr_import.php

```diff
@@ -22,39 +22,34 @@
  * @author  Ajil P M <ajilpm@zhservices.com>
  * @link    http://www.open-emr.org
  */
```

Comment:
since we are here, lets simplify this header:
``````
/**
 * interface/patient_file/ccr_import.php Upload screen and parser for the CCR XML.
 *
 * Functions to upload the CCR XML and to parse and insert it into audit tables.
 *
 * @package OpenEMR
 * @link    http://www.open-emr.org
 * @author  Eldho Chacko <eldho@zhservices.com>
 * @author  Ajil P M <ajilpm@zhservices.com>
 * @author  YOUR AUTHOR INFO HERE
 * @copyright Copyright (c) 2013 Z&H Consultancy Services Private Limited <sam@zhservices.com>
 * @copyright Copyright (c) 2017 YOUR AUTHOR INFO HERE
 * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
 */
``````